app/graphql/schema/comms-message/type-definitions.js

```diff
@@ -0,0 +1,68 @@
+const commsMessageTypeDefs = `#graphql
+# Scalars for custom data types
```

Comment:
Initially we though that this would allow us to see those comments(descriptions) within gql playground/introspection, however this is not the case, so we are on the way to redo them with """ notion

Reply:
changed to have '"" notion